site/ko/swift/python_interoperability.ipynb

```diff
@@ -0,0 +1,677 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Te7sNNx9c_am"
+   },
+   "source": [
+    "## 파이썬 내장객체\n",
+    "\n",
+    "Python의 전역 인터페이스를 활용해 Python 내장 객체에 진입합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 68
+    },
+    "colab_type": "code",
+    "id": "jpcOByipc75O",
+    "outputId": "483be552-f2df-498c-e3c6-b0946040108d"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'int'>\r\n",
+      "3\r\n",
+      "6\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "// `Python.builtins`은 모든 파이썬 내장객체의 딕셔너리입니다.\n",
+    "_ = Python.builtins\n",
+    "\n",
+    "// 파이썬 내장 객체를 시도합니다.\n",
+    "print(Python.type(1))\n",
+    "print(Python.len([1, 2, 3]))\n",
+    "print(Python.sum([1, 2, 3]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "H2wwUL1tY3JX"
+   },
+   "source": [
+    "## 파이썬 모듈 임포트\n",
+    "\n",
+    "`Python.import`를 사용하여 `Python` 모듈을 임포트합니다. 이것은 파이썬의 `import` 키워드처럼 동작합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 68
+    },
+    "colab_type": "code",
+    "id": "XrZee8n3Y17_",
+    "outputId": "dc97f143-8af5-42bf-f5cd-2f4c54cc9bb1"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<module 'numpy' from '/usr/local/lib/python3.6/dist-packages/numpy/__init__.py'>\r\n",
+      "[[1. 1. 1.]\r\n",
+      " [1. 1. 1.]]\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "let np = Python.import(\"numpy\")\n",
+    "print(np)\n",
+    "let zeros = np.ones([2, 3])\n",
+    "print(zeros)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "hQvza3dUXlr0"
+   },
+   "source": [
+    "안전하게 패키지를 가져오기 위해 예외처리 함수 `Python.attemptImport`를 사용합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 34
+    },
+    "colab_type": "code",
+    "id": "QD-uQGuaXhrM",
+    "outputId": "6dec8823-f4f2-4dcb-da7f-4ee07558f1ef"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "nil\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "let maybeModule = try? Python.attemptImport(\"nonexistent_module\")\n",
+    "print(maybeModule)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Qej_Z6V3mZnG"
+   },
+   "source": [
+    "## `numpy.ndarray`로 변환\n",
+    "\n",
+    "The following Swift types can be converted to and from `numpy.ndarray`:\n",
+    "- `Array<Element>`\n",
+    "- `ShapedArray<Scalar>`\n",
+    "- `Tensor<Scalar>`\n",
+    "\n",
```

Comment:
Line 487-491 is the English document and should be removed.

Reply:
I remove them